fatiando/gravmag/transform.py

```diff
@@ -112,6 +114,59 @@ def tga(x, y, data, shape):
     return res
 
 
+def mstde(x, y, data, shape):
+    """
+    Multi-Scale Tilt Depth Estimation
+
+    Implements the method of van Buren (2013).
+
+    An empirical method for attempting to approximate the depth of the source
+    of a magnetic anomaly, based on altering tilt and continuation of the
+    magnetic field intensity.
+
+    **References**
+
+    Van Buren, Reece. 2013. “Multi-Scale Tilt Depth Estimation.” MSc Thesis,
+    Johannesberg: University of the Witwatersrand.
+    URI: http://mobile.wiredspace.wits.ac.za/bitstream/handle/10539/14011/MSc_Dissertation_RvanBuren_2013.pdf.
+    """
+
+
+def tilt(x, y, data, shape):
+    """
+    Calculates the magnetic tilt, as defined by Miller and Singh (1994):
+
+    tilt(f) = tan^{-1}(\\frac{\\frac{df}{dz}}{\\sqrt{(\\frac{df}{dx})^2}})
+
+    .. warning::
+
+        If the data is not in SI units, the derivatives will be in
+        strange units! I strongly recommend converting the data to SI
+        **before** calculating the derivative (use one of the unit conversion
+        functions of :mod:`fatiando.utils`). This way the derivative will be in
+        SI units and can be easily converted to what unit you want.
```

Comment:
It doesn't really matter what the units of the data are because the tilt angle will be in radians anyway, right? The ratio df/dz / df/dh will be unitless.